packages/composer/drupal/cypress/src/CachedInstallation.php

```diff
@@ -410,14 +439,19 @@ protected function restoreCache($cacheDir) {
    *
    * @param string $cacheDir
    *   The cache directory to persist.
+   *
+   * @return void
    */
   protected function writePersistentCache($cacheDir) {
     $files = [];
     $finder = new Finder();
     $finder->files()->in($cacheDir);
     $finder->ignoreDotFiles(FALSE);
     foreach ($finder as $file) {
-      $files[$file->getRelativePath() . '/' . $file->getBasename()] = $file->getRealPath();
+      $realpath = $file->getRealPath();
+      if ($realpath !== FALSE) {
+        $files[$file->getRelativePath() . '/' . $file->getBasename()] = $realpath;
+      }
```

Comment:
At first the code was
```php
      if ($realpath === FALSE) {
        throw new \Exception('Cannot get realpath for a file: "' . $file->getRelativePath() . '/' . $file->getBasename() . '".');
      }
      $files[$file->getRelativePath() . '/' . $file->getBasename()] = $realpath;
```
But then I've got `Exception: Cannot get realpath for a file: "/settings.php".`

So I changed it to the current less strict version.

Not sure if it's right though.